src/test/java/io/strimzi/kafka/bridge/http/services/ConsumerService.java

```diff
@@ -33,6 +33,8 @@
 
     static final Logger LOGGER = LogManager.getLogger(ConsumerService.class);
 
+    private static final int POLL_TIMEOUT = 4000;
```

Comment:
I guess you get this value from your investigation. Any idea why it is so high?

Reply:
Tbh I just set a reasonable value higher than 1000ms. Anyway, it's not there to wait for 4 secs, the poll returns even before when rebalance is done and records are available. This is a max value, not a time to wait in each test.

Reply:
I see. It is passing so far, well done!